hasher-matcher-actioner/webapp/src/pages/ContentDetails.jsx

```diff
@@ -34,15 +40,25 @@ export default function ContentDetails() {
 
   // TODO fetch actions once endpoint exists
   useEffect(() => {
-    setActions([]);
+    fetchContentActionHistory(id).then(result => {
+      if (result && result.action_history) {
+        setActionHistory(result.action_history);
+      }
+    });
+  }, []);
```

Comment:
Is there a reason you have `, []` on this line and at line 54?

Reply:
Unless we specify what attribute changes to useEffect() on, react gets into an infinite loop. `fetchContentActionHistory` changes state, causing the effect to be triggered again which changes state which .....

The `[]` tells react that useEffect should only be called once. Normally, you'd specify which attributes (state, props) you want to react to, but in this case, we want to react to nothing.